doc_src/examples/data_io.rst

```diff
@@ -74,21 +74,4 @@ Create Dataframe from Tamr Dataset
 Validate a Dataframe before upserting records to a Tamr Dataset
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .. literalinclude:: ../../examples/snippets/data_io/dataframe/validate_dataframe.py
- :language: python
-
-
-CSV
-------------------------------------------------------------------------
-
-Export CSV from Tamr Dataset to a designated filepath
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-.. literalinclude:: ../../examples/resources/conf/dataset.config.yaml
-  :language: yaml
-.. literalinclude :: ../../examples/scripts/data_io/csv/export_from_dataset.py
-  :language: python
-
-
-Export CSV from taxonomy to a designated filepath
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-.. literalinclude:: ../../examples/snippets/data_io/csv/export_from_taxonomy.py
-  :language: python
\ No newline at end of file
+ :language: python
```

Comment:
It looks like the CSV block here got dropped accidentally.  Let me know if it would help to go over any git issues together.